content/collections/workflow/en/experiment-learnings.md

```diff
@@ -13,59 +13,119 @@ You’ve designed your experiment, rolled it out to your users, and given them e
 
-2. At the top of the *Analysis* section is an overview of how your experiment performed, broken down by metric and variant. Below that is the experiment's **exposure definition:** how many variants were shown, what the primary metric was, and what the **exposure event** was. This is the event users will have to fire before being included in an experiment.  
-  
-    {{partial:admonition type='note'}}
-    The exposure event is **not the same thing** as the assignment event. If, for example, you’re running an experiment on your pricing page, a user might be evaluated on the home page for the experiment—but if they don’t visit the pricing page, they'll never actually be exposed to it. For that reason, this user should not be considered to be part of the experiment.  
-    {{/partial:admonition}}
+{{partial:admonition type="note" heading="Availability"}}
+Summary is available to organizations with access to Experiment who have recommendations enabled.
+{{/partial:admonition}}
+
+The Summary card describes your experiment's hypothesis and lets you know if it's reached statistical significance. 
+
+{{partial:admonition type="note" heading="Statisical significance and Amplitude"}}
+Amplitude considers an experiment to be **statistically significant** when Amplitude can confidently say that the results are unlikely to have occurred due to random chance. More technically, it’s when Amplitude rejects the null hypothesis. That may sound subjective, but it’s grounded solidly in statistics. Statistical significance relies on a variant’s **p-value**, which is a value that represents the likelihood that your results occurred by chance. A lower p-value means your results are probably not random, and there's evidence to support your hypothesis. If this value drops below a threshold, Amplitude considers the experiment to be statistically significant.
+{{/partial:admonition}}
+
+The Summary card displays a badge labeled *Significant* if the experiment reached statistical significance, and a badge labeled *Not Significant* if it didn't. This card can display several badges at once:
+
+* *Inconclusive*: the test was inconclusive for the primary metric.
+* *Above Goal* or *Below Goal:* the primary metric's mean was either **above** or **below** its goal depending on the direction of the test (increase = above, decrease = below).
+* *Above Control* or *Below Control:* the primary metric's mean was either **above** or **below** the control's mean, depending on the direction of the test (increase = above, decrease = below). These badges are only relevant to stat sig results.
+
+![summary.png](/docs/output/img/workflow/summary-png.png)
+
+
+## Analysis card
+
+At the top of the Analysis card is an overview that explains how your experiment performed, broken down by metric and variant. Below that, a collection of experiment results charts, which you can analyze by metric, display information about:
+
+* Confidence intervals
+* Cumulative exposure*
+* Event totals
```

Comment:
This chart is not always for event totals. For example the metric can be prop sum or uniques or anything